tests/manager/manager-ctor_error-003.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): read preference options of the wrong type will throw an InvalidArgumentException exception
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$invalidOptions = array(
+    [],
+    123,
+    2.5,
+    0x00,
+    null,
+    true
+);
```

Comment:
Since `manager-ctor-read_preference-003.phpt` in its current state will no longer apply, I'd suggest deleting the file in git. The new test's filename will be `manager-ctor-read_preference-error-004.phpt` (4 is next in the sequence) and can be based on the existing contents of `manager-ctor-read_preference-003.phpt`.

I don't think it matters if you delete `manager-ctor-read_preference-003.phpt` and add the new test or use `git mv` to rename the test. Since you'll be modifying the original test anyway, git will decide the change is sufficient to consider them two different files or a rename with modifications.